### PR TITLE
Initial implementation of lpython builtin module

### PR DESCRIPTION
**Pull request by namannimmo10**
_Originally taken from https://github.com/lcompilers/lpython/issues/1#issuecomment-1004281032_

### CODE REVIEW

**Comment by certik** [comment]
Now we need to import this automatically when `chr` is encountered in AST to ASR and call it.

**Comment by certik** [comment]
That looks good. The tests fail, so that needs to be fixed.

**Comment by certik** [comment]
The `load_module` function has to accept the main (global) `TranslationUnit` and it will use it to:

* on "import A" it will lookup the global TranslationUnit for a module A and use it (construct ExternalSymbols from it), otherwise it will recursively call itself to import `A`, and then use it
* Once it's done importing a given module, it will save it to the global TranslationUnit

File: src/runtime/lpython_builtin.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from ltypes import i32
+from sys import exit
```
[the remark this answers is not in the document]
Comment (namannimmo10) [reply src/runtime/lpython_builtin.py:2]: Once we do that, it actually executes the compile-time `chr` and `ord` built-in functions.

**Comment by certik** [comment]
It should work now. The strings in `ord` and `chr` are not always implemented in the LLVM backend, so we can't actually run it, but the ASR looks pretty good. I need to implement some intrinsic function that actually works in LLVM, then we can properly test it.

**Comment by certik** [comment]
I implemented `abs()` to load our own function. It seems to work now. The compile time value is not implemented yet, we'll do it in a subsequent PR. Finally, we really have to refactor this to follow LFortran's design.

**Comment by namannimmo10** [comment]
I think this should be ready to go in -- we can complete the remaining tasks in upcoming PRs.

**Comment by certik** [comment]
This approach now needs to be refactored like in LFortran.

**Pull request merged**